trie-db/src/trie_codec.rs

```diff
@@ -34,6 +34,8 @@ use crate::{
 };
 use hash_db::{HashDB, Prefix};
 
+const OMMIT_VALUE_HASH: crate::node::Value<'static> = crate::node::Value::Inline(&[]);
```

Comment:
```suggestion
const OMIT_VALUE_HASH: crate::node::Value<'static> = crate::node::Value::Inline(&[]);
```

You got one `M` too many here. (:

Reply:
Thanks :)